Mathlib/MeasureTheory/MeasurableSpace/CountablyGenerated.lean

```diff
@@ -459,4 +328,168 @@ lemma measurableSet_countablePartitionSet (n : ℕ) (a : α) :
     MeasurableSet (countablePartitionSet n a) :=
   measurableSet_countablePartition n (countablePartitionSet_mem n a)
 
+/-! ### Countably generated spaces and separating sets -/
```

Comment:
You're simultaneously moving things around, and changing some stuff. Can you explain what has changed and what is just moving around, to help the review?